### PR TITLE
Automatically port Reflection.Metadata A-D APIs

### PR DESCRIPTION
**Pull request by carlossanlop**
Area owners: @tmat, @nguerrera

### CODE REVIEW

File: xml/System.Reflection.Metadata/Document.xml [hunk 1]
```diff
@@ -23,8 +23,16 @@
...
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>The source document in the debug metadata.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+See also https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md#document-table-0x30.
```
Comment (Thraka) [inline xml/System.Reflection.Metadata/Document.xml]: Some markdown parsers will not encode inline https as a link. It's better to specify it explicitly. I'm also editing the text here to make it more user friendly.

```suggestion
For more information, see [Portable PDB v1.0: Format Specification](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md#document-table-0x30).
```

Comment (carlossanlop) [reply xml/System.Reflection.Metadata/Document.xml]: Thank you very much @Thraka for taking a look. I'll apply your suggestion.

**Comment by Thraka** [comment]
I'll sign off, but do you still want a technical review from @tmat Tomas Matousek FTE, @nguerrera Nick Guerrera FTE ?

**Comment by carlossanlop** [comment]
Thank you, @Thraka. Ideally we would like area owners to review as well (@tmat and @nguerrera), although since these were automatically ported triple slash comments from source, I'm not too concerned about the content. Your review was more important (proper language use).
Since you already reviewed and signed off, and the build passed with no warning, feel free to merge it whenever you want, @Thraka / @mairaw.

**Pull request merged**